tests/flytekit/unit/cli/pyflyte/test_run.py

```diff
@@ -274,9 +274,9 @@ def test_list_default_arguments(wf_path):
 )
 
 ic_result_4 = ImageConfig(
```

Comment:
Question:
Are these tags calculated by [`calculate_hash_from_image_spec`](https://github.com/danpf/flytekit/blob/44f42cfa4f656ed67a525a2611b510dab0d90981/flytekit/image_spec/image_spec.py#L169)?

Reply:
yes, it changed because the interface changed (dockerfile... etc)